akka-docs/src/main/paradox/stream/operators/Compression/deflate.md

```diff
@@ -0,0 +1,34 @@
+# Compression.deflate
+
+Creates a flow that deflate-compresses a stream of ByteString.
+
+@ref[Actor interop operators](../index.md#actor-interop-operators)
```

Comment:
I think this operator doesn't really belong in this section - can we find another section to add this operator to? Or perhaps we should introduce a new one?